src/Paprika.Tests/Data/NibblePathTests.cs

```diff
@@ -1,14 +1,129 @@
 using System.Buffers.Binary;
 using FluentAssertions;
 using NUnit.Framework;
+using Paprika.Crypto;
 using Paprika.Data;
+using Paprika.Utils;
 
 // ReSharper disable HeapView.BoxingAllocation
 
 namespace Paprika.Tests.Data;
 
 public class NibblePathTests
 {
+    [Test]
+    public void Get_Singles()
+    {
+        // Expected values
+        byte[] expectedSingles =
+        [
+            0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
+            0x88, 0x99, 0xAA, 0xBB, 0xCC, 0xDD, 0xEE, 0xFF
+        ];
+
+        // Get the actual values from the NibblePath.Singles property
+        var actualSingles = NibblePath.SinglesForTests;
```

Comment:
I don't understand this test. It just exposes singles?

Reply:
Yes, this was for covering this under tests to increase overall coverage 😅